tests/scenario/modules/assets/asset-history.test.ts

```diff
@@ -35,6 +35,75 @@ describe("DeviceController: receiveMeasure", () => {
     expect(result.hits[0]._source.event.metadata).toBeUndefined();
   });
 
+  it("should historize asset for each measurements of the same measure received in non-chronological order", async () => {
+    await sendDummyTempPayloads(sdk, [
+      {
+        measurements: [
+          {
+            deviceEUI: "linked1",
+            temperature: 40.2,
+            measuredAt: 1680096420000, // 1:27:00 PM UTC
+          },
+          {
+            deviceEUI: "linked1",
+            temperature: 41.2,
+            measuredAt: 1680096360000, // 1:26:00 PM UTC
+          },
+          {
+            deviceEUI: "linked1",
+            temperature: 42.2,
+            measuredAt: 1680096300000, // 1:25:00 PM UTC
+          },
+        ],
+      },
+    ]);
+    await sdk.collection.refresh("engine-ayse", "assets-history");
+
+    const result = await sdk.document.search<AssetHistoryContent>(
+      "engine-ayse",
+      "assets-history",
+      { sort: { timestamp: "desc" } }
```

Comment:
Since asset historization promises are parallelized, they are not always created in the right order.
We added the property timestamp in order to retrieve those documents in the right order..

Reply:
Done